examples/react-apollo-live/.env.example

```diff
@@ -0,0 +1,3 @@
+# Refer to the README on how to fill these variables
+VITE_GRAFBASE_API_URL=
+VITE_CLERK_FRONTEND_API=
```

Comment:
We should maybe include a comment at the top of this file that the Clerk URL must be the full absolute URL, so we can remove the protocol from the `schema.graphql` and only rely on the true ENV value threre.

Reply:
Should we? I find it easier to copy-paste exactly how Clerk provides URL in their dashboard

Also, note that `VITE_CLERK_FRONTEND_API` is used in ClerkProvider without URL protocol, so we would have two versions of the same thing 
- `VITE_CLERK_FRONTEND_API` without protocol 
- `CLERK_ISSUER_URL` with protocol

Reply:
I think we should yeah. They are separate `.env` files after all, so you'd have the same duplication anyways.